packages/kit/src/views/Market/MarketHomeV2/layouts/DesktopLayout.tsx

```
@@ -1,6 +1,6 @@
 import { useCallback, useMemo } from 'react';
 
-import { Carousel, Tabs, YStack } from '@onekeyhq/components';
+import { Carousel, Tabs, useTabContainerWidth, YStack } from '@onekeyhq/components';
```

ESLint

[error] Replace `·Carousel,·Tabs,·useTabContainerWidth,·YStack·` with `⏎··Carousel,⏎··Tabs,⏎··useTabContainerWidth,⏎··YStack,⏎`

(prettier/prettier)